src/TypeScript/Analyzer/AnalyzerBase.cs

```diff
@@ -121,7 +124,7 @@ protected virtual void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-                    eslintBridgeAnalyzer.Dispose();
+                    eslintBridgeAnalyzer.Value.Dispose();
```

Comment:
Depending on how the rest of the code is structured, there's a slight chance that `Value` hasn't been accessed before, so calling `Value.Dispose()` would create the object. It would be safer to check `ValueCreated` on the Lazy first.